src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo-link {
+  line-height: 0;
+  margin: 10px 0;
+}
+
+.nav-bar {
+  display: flex;
+  height: 100;
+  list-style: none;
+}
+
+.nav-bar__list {
+  white-space: nowrap;
+}
+
+.nav-bar__links {
+  position: relative;
+  display: inline-block;
+  margin-left: 20px;
+  line-height: 60px;
+  height: 60px;
+  font-size: 12px;
+  color: #000;
+  text-transform: uppercase;
+  text-decoration: none;
+  transition: color 0.3s;
+}
+
+.first-link {
+  margin-left: 0;
+}
+
+.nav-bar__links.is-active::after {
```

Comment:
do not increase specificity, just use is-active::after
and move it below the
.is-active {
  color: #00acdc;
}